src/utils/getAllArticles.js

```diff
@@ -0,0 +1,12 @@
+import axios from "axios";
```

Comment:
Your utils don't need to be in their own files. What if we have 100+ api calls? We can have a utils file where they all live :)

Reply:
noted